functions/bfc-functions.php

```diff
@@ -182,3 +182,25 @@ function bfc_filter_bbp_reply_admin_links( $array, $r_id ) {
 
 // add the filter
 add_filter( 'bbp_get_reply_admin_links', 'bfc_filter_bbp_reply_admin_links', 10, 2 );
+
+//removes 'private' and protected prefix for forums
+function remove_private_title($title) {
+	return '%s';
+	}
```

Comment:
This curly bracket should go back one indent. Same for line 193.

Reply:
Done.